tests/sanity_helpers.py

```diff
@@ -52,23 +57,42 @@ def create_resources(self, pvc_factory, pod_factory, run_io=True):
             pvc_obj = pvc_factory(interface)
             self.pvc_objs.append(pvc_obj)
             self.pod_objs.append(pod_factory(pvc=pvc_obj))
+            pvc_obj = pvc_factory(interface)
+            self.pvc_objs.append(pvc_obj)
+            self.dc_pod_objs.append(dc_pod_factory(pvc=pvc_obj))
         if run_io:
             for pod in self.pod_objs:
                 pod.run_io('fs', '1G')
+            for dc_pod in self.dc_pod_objs:
+                dc_pod.run_io('fs', '1G')
             for pod in self.pod_objs:
                 get_fio_rw_iops(pod)
+            for dc_pod in self.dc_pod_objs:
+                get_fio_rw_iops(dc_pod)
 
-    def delete_resources(self):
+    def delete_resources(self, ignore_delete=False):
         """
         Sanity validation - Delete resources (FS and RBD)
 
+        Args:
+            ignore_delete (bool): skip deletion of pod if deleted already
```

Comment:
Why is this needed? Wouldn't it be better to just check if pod exists before deleting it? With implementation from this PR, there can be missed some unexpected errors of pod deletion that might not be intended behaviour as this option ignores all command errors for all pod objects created in `Sanity` class.

Reply:
Reason of this param is when doing node replacement some nginx pod will get deleted so at script teardown fails for deleting pod obj and script will never get a pass

Reply:
Wouldn't it be better to just check that the pod exists and if not then don't initiate delete? It would reduce a possibility that we will miss an error that we don't want to miss during deleting all pods.

Reply:
Okay this can be done will update in next commit